src/components/dl-displayer-controller.tsx

```diff
@@ -13,14 +13,16 @@ import { DistributionListDisplayer } from './dl-displayer';
 import { EmptyDisplayer } from './empty-displayer';
 import { useGetDistributionList } from '../hooks/use-get-distribution-list';
 import { useGetDistributionListMembers } from '../hooks/use-get-distribution-list-members';
-import { useActiveItem } from '../hooks/useActiveItem';
 
-export const DLDisplayerController = (): React.JSX.Element => {
-	const { activeItem } = useActiveItem();
+type DLDisplayerControllerProps = {
+	id: string | undefined;
+};
+
+export const DLDisplayerController = ({ id }: DLDisplayerControllerProps): React.JSX.Element => {
 	const [t] = useTranslation();
 	const dlToLoad = useMemo<Parameters<typeof useGetDistributionList>[0] | undefined>(
-		() => (activeItem ? { id: activeItem } : undefined),
-		[activeItem]
+		() => (id ? { id } : undefined),
+		[id]
 	);
 	const distributionList = useGetDistributionList(dlToLoad);
```

Comment:
Probably an improvement could be to make useGetDistributionList to receive string | undefined instead of receving any item which has id or email in it. I am not a fan of the line 24

Reply:
The problem of passing only a string is that we can't know if that string is the email or the ID. On network level, we need to know if we have the id o the email to set a different param in the request. Do you have a proposal on how to keep the 2 values meaningful when passed down to the hook/network?

Reply:
Ok, I saw the only usage of that hook was with the id, I didn't dig deep enough to see the API call can receive both keys. I think it can pass as it is then. Can't think of a better way to manage it